packages/common/src/ui/components/inputs/Switch/Switch.tsx

```diff
@@ -50,25 +68,28 @@ export const Switch: React.FC<SwitchProps> = ({
     padding: isSmall ? '1px' : '6px 12px',
     '& .MuiSwitch-switchBase': {
       paddingLeft: '3px',
-
       right: 'auto', // emotion is setting this and making a mess
     },
+    '& .MuiSwitch-thumb': {
+      color: disabled ? 'gray.light' : 'gray.main',
+    },
     '& .Mui-checked .MuiSwitch-thumb': {
       color: 'inherit',
     },
-    '& .MuiSwitch-thumb': {
-      color: 'gray.dark',
-    },
     '& .MuiSwitch-track': {
       borderWidth: 1,
       borderStyle: 'solid',
-      borderColor: 'border',
+      borderColor: 'gray.main',
       backgroundColor: '#fff',
+      borderRadius: isSmall ? '11px' : '13px',
       transition: (theme: Theme) =>
         theme.transitions.create(['background-color'], {
           duration: 500,
         }),
     },
+    '& .Mui-checked ~ .MuiSwitch-track': {
```

Comment:
sibling selector - was using `checked` but that only works for controlled components!